.env.example

```diff
@@ -11,7 +11,7 @@ DB_PASSWORD=secret
 
 CACHE_DRIVER=file
 SESSION_DRIVER=file
-QUEUE_DRIVER=sync
+QUEUE_DRIVER=database
```

Comment:
I don't think we should recommend the database queue if we do get people to use a queue.